### PR TITLE
Issue #9548: add code example of AST for TokenTypes.OBJBLOCK

### PR DESCRIPTION
**Pull request by CryptoSingh1337**
Closes: #9548 
image

**Successfully compiled**
image

**Test.java**
`λ cat Test.java`
```
class Test {}
```
**AST**
`λ java -jar checkstyle-8.41-all.jar -t Test.java`
```
CLASS_DEF -> CLASS_DEF [1:0]
|--MODIFIERS -> MODIFIERS [1:0]
|--LITERAL_CLASS -> class [1:0]
|--IDENT -> Test [1:6]
`--OBJBLOCK -> OBJBLOCK [1:11]
    |--LCURLY -> { [1:11]
    `--RCURLY -> } [1:12]
```
### Expected update for javadoc
```
CLASS_DEF -> CLASS_DEF
|--MODIFIERS -> MODIFIERS
|--LITERAL_CLASS -> class
|--IDENT -> Test
`--OBJBLOCK -> OBJBLOCK
    |--LCURLY -> {
    `--RCURLY -> }
```

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -73,6 +73,18 @@
      * annotation and enum constant declarations.
      * Also, object blocks are children of the new keyword when defining
      * anonymous inner types.
+     * <p>For example:</p>
+     * <pre>class Test {}</pre>
```
Comment (timurt) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:77]: please, put tags and content on separate lines

**Review by timurt — CHANGES_REQUESTED** [review]

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 2]
```diff
@@ -73,6 +73,24 @@
      * annotation and enum constant declarations.
      * Also, object blocks are children of the new keyword when defining
      * anonymous inner types.
+     * <p>
+     *     For example:
```
Comment (timurt) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:77]: this one can be written in one line, I meant `<pre>` tag

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 3]
```diff
@@ -73,6 +73,24 @@
...
+     * </p>
+     * <pre>
+     *     class Test {}
```
Comment (timurt) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:80]: please delete preceding spaces

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 4]
```diff
@@ -73,6 +73,24 @@
...
+     * </pre>
+     * <p>
+     *     parses as:
```
Comment (timurt) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:83]: same as for `For example:` section, should be on the same line

you can use https://github.com/checkstyle/checkstyle/pull/9378/files as an example

**Review by timurt — CHANGES_REQUESTED** [review]

**Review by romani — APPROVED** [review]

**Pull request merged**